datacenter/ucp/2.1/guides/admin/configure/external-auth/index.md

```diff
@@ -1,63 +1,77 @@
 ---
-description: Learn how to integrate UCP with an LDAP service, so that you can manage
-  users from a single place.
-keywords: LDAP, authentication, user management
-title: Integrate with LDAP
+description: Learn how to integrate UCP with an LDAP service, so that you can
+  manage users from a single place.
+keywords: LDAP, directory, authentication, user management
+title: Integrate with an LDAP Directory
 ---
 
 Docker UCP integrates with LDAP services, so that you can manage users from a
-single place.
+single place. Many enterprise organizations have an existing directory service
+where people and groups are managed. UCP allows you to sync people and groups
+from your organization's directory to automatically provision users and team
+members for Docker UCP and Docker Trusted Registry.
 
 ## Configure the LDAP integration
 
-To configure UCP to authenticate users using an LDAP service, go to
-the **UCP web UI**, navigate to the **Settings** page, and click the **Auth**
-tab.
+To configure UCP to provision and authenticate users using an LDAP directory,
```

Comment:
nitpick but the word "provision" in your head is very clear but might not be for non-expert users. Maybe "To configure UCP to synchronize user information and authenticate them using ..."